src/App.js

```diff
@@ -1,10 +1,25 @@
-import './App.css';
+import { useState } from 'react';
+import Display from './components/Display';
 import Calculator from './components/Calculator';
 
+import './App.css';
+
 function App() {
+  const [calculatorDisplay, setCalculatorDisplay] = useState({
+    next: 0,
+    total: null,
+    operation: null,
+  });
+
+  const displayHandler = (symbol) => {
+    setCalculatorDisplay(symbol);
+    // console.log(symbol);
+  };
```

Comment:
- [ ] Kindly remove commented unused code from your codebase as this makes your code not clean and professional. Please remove all used commented codes 👍🏽